packages/pdfkit/src/mixins/acroform.js

```diff
@@ -46,6 +46,10 @@ export default {
    * this method to set the default font.
    */
   initForm() {
+    if (!!this._root.data.AcroForm) {
+      // Form is already initialized
+      return this;
+    }
```

Comment:
Fixes an issue when using `<Form />` twice on e.g. two pages.

Reply:
Why not check this inside the render package itself? 
That way, it would not interfere with this issue: https://github.com/diegomura/react-pdf/issues/2613

Reply:
And what about removing the `<Form />` component? Its sole purpose is initializing on the document. Shouldn't it be a property on the `<Document />` then, or maybe even just a check inside the form components?

Reply:
Hey @PhilippBloss, thanks for the input. I did not find the place where I could add this to `<Document />`. But I added it to form components instead now: https://github.com/diegomura/react-pdf/pull/2845/commits/7162e6f4b1bb01f72c26a0a052ef4ac76e8b7971.